### PR TITLE
Initial YouTube Search Service Tests

### PR DESCRIPTION
**Pull request by deini**
@alexmckenley @malucky Initial YouTube Search Service Tests please review

### CODE REVIEW

**Comment by malucky** [comment]
are you also testing the /videos endpoint?
everything else LGTM

**Comment by deini** [comment]
Yeah this is testing both endpoints since the .search() calls both of them. It also kinda tests the regex that converts the time to seconds.

File: public/_common/services/providers/thirdPartyProviders/youtube/player/youtubePlayer.service.js [hunk 1]
```diff
@@ -55,7 +55,7 @@
 
     /**
      * Seek the youtube player to a specific second
-     * @param {integer} second The time in seconds to a specific part of the video
+     * @param {Number} second The time in seconds to a specific part of the video (integer)
```
Comment (alexmckenley) [inline public/_common/services/providers/thirdPartyProviders/youtube/player/youtubePlayer.service.js]: didnt we say that all types would be lowercase?

Comment (deini) [inline public/_common/services/providers/thirdPartyProviders/youtube/player/youtubePlayer.service.js]: Yes, will fix :)

On Tue, Oct 14, 2014 at 11:53 AM, Alex Taylor notifications@github.com
wrote:

> In
> public/_common/services/providers/thirdPartyProviders/youtube/player/youtubePlayer.service.js:
> 
> > @@ -55,7 +55,7 @@
> > 
> > ```
> >  /**
> >   * Seek the youtube player to a specific second
> > ```
> > -     \* @param {integer} second The time in seconds to a specific part of the video
> > -     \* @param {Number} second The time in seconds to a specific part of the video (integer)
> 
> didnt we say that all types would be lowercase?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/ammoio/ammo/pull/339/files#r18850314.

## 

Daniel Almaguer

File: public/_common/services/providers/thirdPartyProviders/youtube/player/youtubePlayer.service.js [hunk 2]
```diff
@@ -193,4 +193,4 @@
       }
     }
   }
-})();
+}());
```
Comment (alexmckenley) [inline public/_common/services/providers/thirdPartyProviders/youtube/player/youtubePlayer.service.js:196]: Did intelliJ tell you to do this? It have me a lint error the other day for this, but I'm not actually sure if its right.

Comment (deini) [inline public/_common/services/providers/thirdPartyProviders/youtube/player/youtubePlayer.service.js:196]: Yeah my IDE suggested me this way and I confirmed it with the lord http://javascript.crockford.com/code.html

**Comment by deini** [comment]
@alexmckenley 8f1574d

**Comment by alexmckenley** [comment]
I approve :+1:

**Comment by alexmckenley** [comment]
Also :100:  for referencing Douglas Crockford

**Comment by deini** [comment]
That's the only way to convince you :dancer:

On Thu, Oct 16, 2014 at 12:09 AM, Alex Taylor notifications@github.com
wrote:

> Also [image: :100:] for referencing Douglas Crockford
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/ammoio/ammo/pull/339#issuecomment-59322370.

## 

Daniel Almaguer

**Comment by deini** [comment]
Squash & Merge

**Pull request merged**